Tools/XcodeCapp/XcodeCapp/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSUserNotificationAlertStyle</key>
+	<string>alert</string>
```

Comment:
@didierkorthoudt You two lines have sneaked in here too. 😄

Reply:
Strange… Should be in my own branch… I think I'll have to rebase it !.. OK, I adapt.